src/config.ts

```diff
@@ -16,6 +16,15 @@ export const MARKETPLACE_SALEOR_APPS_JSON_PATH =
 export const MARKETPLACE_APP_TEMPLATE_GALLERY_PATH =
   window.__SALEOR_CONFIG__.APP_TEMPLATE_GALLERY_PATH;
 
+export const AppsConfig = {
+  marketplaceUrl:
+    window.__SALEOR_CONFIG__.APPS_MARKETPLACE_URL ||
+    "https://marketplace-gray.vercel.app/api/v2/saleor-apps",
+  tunnelUrlKeywords: window.__SALEOR_CONFIG__.APPS_TUNNEL_URL_KEYWORDS?.split(
```

Comment:
please ensure new envs are documented

Reply:
Added to `docs/configuration.md` + `.github/PULL_REQUEST_TEMPLATE.md`